actors/miner/src/state.rs

```diff
@@ -1230,6 +1231,12 @@ pub struct MinerInfo {
     /// Optional worker key to update at an epoch
     pub pending_worker_key: Option<WorkerKeyChange>,
 
+    // Beneficiary account for receive miner benefits, withdraw on miner must send to this address,
+    // beneficiary set owner address by default when create miner
+    pub beneficiary: Address,
+    pub beneficiary_term: BeneficiaryTerm,
+    pub pending_beneficiary_term: Option<PendingBeneficiaryChange>,
```

Comment:
To reduce confusion in off-chain tools, put these fields at the end of the MinerInfo struct so that all the existing elements retain their index in the serialized tuple.